mozilla_django_oidc_db/tests/utils.py

```diff
@@ -0,0 +1,59 @@
+"""
+Keycloak helpers taken from mozilla-django-oidc-db::tests/utils.py & pytest fixtures.
+
+These help dealing with/stubbing out OpenID Provider configuration.
+
+The Keycloak client ID/secret and URLs are set up for the config in
+docker/docker-compose.keycloak.yml. See the README.md in docker/keycloak/ for more
+information.
+"""
+
+from contextlib import nullcontext
+
+from pyquery import PyQuery as pq
+from requests import Session
+
+KEYCLOAK_BASE_URL = "http://localhost:8080/realms/test/protocol/openid-connect"
+
+
+def keycloak_login(
```

Comment:
Let's remove `keycloak_login` from `tests/utils.py` and use this version in this library's tests as well